docs/migration_guide.md

```diff
@@ -14,61 +14,83 @@ Since the migration might alter the database schema and the settings of `harbor.
+```
 
-    ```sh
-    mv harbor /my_backup_dir/harbor
-    ```
+#### Backup Harbor Data
 
-    Back up database (by default in directory `/data/database`)
+Back up Harbor's current files so that you can roll back to the current version if necessary.
 
-    ```sh
-    cp -r /data/database /my_backup_dir/
-    ```
+```sh
+mv harbor /my_backup_dir/harbor
+```
+
+Back up database (by default in directory `/data/database`)
 
-3. Get the latest Harbor release package from Github:
-   [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases)
+```sh
+cp -r /data/database /my_backup_dir/
+```
 
-4. Before upgrading Harbor, perform migration first.  The migration tool is delivered as a docker image.
+#### Get Package of Latest Harbor Installer
+
+You can download it in this url [https://github.com/goharbor/harbor/releases](https://github.com/goharbor/harbor/releases).
+
+#### Migrating Config file
+
+Before upgrading Harbor, perform migration first. The migration tool is delivered as a docker image.
+
+1. **Online installer** need pull the migration image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
 
-    You can pull the image from docker hub. Replace [tag] with the release version of Harbor (e.g. v1.5.0) in the below command:
     ```sh
     docker pull goharbor/harbor-migrator:[tag]
     ```
 
-    Alternatively, if you are using an offline installer package you can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+    **Offline installer** can load it from the image tarball included in the offline installer package. Replace [version] with the release version of Harbor (e.g. v1.5.0) in the below command:
+
     ```sh
     tar zxf <offline package>
     docker image load -i harbor/harbor.[version].tar.gz
     ```
 
-5. If you are current version is v1.7.x or earlier, i.e. migrate config file from `harbor.cfg` to `harbor.yml`.
+2. **If current Harbor version is v1.7.x or earlier**, You need migrate config file from `harbor.cfg` to `harbor.yml`. So need mount both `harbor.cfg` and `harbor.yml` to run the command.
 
-    **NOTE:** You can find the ${harbor_yml} in the extracted installer you got in step `3`, after the migration the file `harbor.yml`
-    in that path will be updated with the values from ${harbor_cfg}
+    **NOTE:** After the migration the `harbor.yml` file will be updated with the values from `${harbor_cfg}`
 
     ```sh
     docker run -it --rm -v ${harbor_cfg}:/harbor-migration/harbor-cfg/harbor.yml -v ${harbor_yml}:/harbor-migration/harbor-cfg-out/harbor.yml goharbor/harbor-migrator:[tag] --cfg up
     ```
 
-    Otherwise, If your version is 1.8.x or higher, just upgrade the `harbor.yml` file.
+    **If current Harbor version is v1.8.0 or above**, there is no `harbor.cfg` file anymore. So, only `harbor.yml` file need to mount.
```

Comment:
**If your current Harbor version is v1.8.0 or above**, there is no `harbor.cfg` file. So, you only need to mount `harbor.yml`.